### DISCUSSION TITLE
[3.x]: Rebuild all slugs using entry title

### DISCUSSION THREAD

**Discussion by development-lameco** [craftcms/cms#11179] [Q&A] [answered]
Is there a way to rebuild/regenerate all slugs using the entry title?
I've looked through the commands and utilities in the CMS but can't find any option.

**Comment by brandonkelly** [comment, marked as answer]
Yes, you can do this with the [resave](https://craftcms.com/docs/4.x/console-commands.html#resave) console command:

```sh
php craft resave/entries --set slug --to :empty:
```

(When the slugs are set to an empty value, they will automatically get regenerated based on the title.)

**Reply by development-lameco** [reply]
Thanks, I must have missed this or searched for the wrong term in the documentation.
